dialogflow_v2/proto/audio_config_pb2.py

```diff
@@ -639,6 +856,77 @@
 DESCRIPTOR.enum_types_by_name["OutputAudioEncoding"] = _OUTPUTAUDIOENCODING
 _sym_db.RegisterFileDescriptor(DESCRIPTOR)
 
+SpeechContext = _reflection.GeneratedProtocolMessageType(
+    "SpeechContext",
+    (_message.Message,),
+    dict(
+        DESCRIPTOR=_SPEECHCONTEXT,
+        __module__="google.cloud.dialogflow_v2.proto.audio_config_pb2",
+        __doc__="""Hints for the speech recognizer to help with recognition in a specific
+  conversation state.
+  
+  
+  Attributes:
+      phrases:
+          Optional. A list of strings containing words and phrases that
+          the speech recognizer should recognize with higher likelihood.
+          This list can be used to: \* improve accuracy for words and
+          phrases you expect the user to say, e.g. typical commands for
+          your Dialogflow agent \* add additional words to the speech
+          recognizer vocabulary \* ...  See the `Cloud Speech
+          documentation <https://cloud.google.com/speech-to-
+          text/quotas>`__ for usage limits.
+      boost:
```

Comment:
I need to check, but Speech v1 removed `boost` from the beta and I think Dialogflow relies on that.